### PR TITLE
{geo}[GCCcore/11.2.0] SNAP-ESA v9.0.0 + SNAP-ESA-python v9.0 w/ Python 2.7.18

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

requires:
* https://github.com/easybuilders/easybuild-easyconfigs/pull/13198

for the `SNAP-ESA-8.0-response.varfile` it includes

had to tweak the sanity check command a bit to avoid that it hangs...)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3130.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/6ce6e75449d86dd199f3f8d30fe998ba for a full test report.

File: easybuild/easyconfigs/s/SNAP-ESA/SNAP-ESA-9.0.0-Java-1.8.eb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+patches = [(local_varfile, '.')]
+checksums = [
+    {'esa-snap_all_unix_9_0_0.sh': 'ad6ef8c679c80c58660fe23ab5fb4974f9cd3ff4b119ad8e7e40dc79dbe03896'},
+    {'SNAP-ESA-8.0-response.varfile': '5ea98e3376bb3df52b9c4c99ab4986015296b815021e16486b58e4aad58e21a4'},
+]
+
+# The installation is executed with the bundled JRE 1.8.0_242 (Zulu)
+# at runtime we switch it to AdoptOpenJDK, which is known to be more reliable for SNAP-ESA
+dependencies = [
+    ('Java', '1.8'),
```
Comment (lexming) [inline easybuild/easyconfigs/s/SNAP-ESA/SNAP-ESA-9.0.0-Java-1.8.eb:36]: The current installation instructions say that it needs Java/11. Is it worth the upgrade?
See https://senbox.atlassian.net/wiki/spaces/SNAP/pages/10879039/How+to+build+SNAP+from+sources

Comment (Micket) [reply easybuild/easyconfigs/s/SNAP-ESA/SNAP-ESA-9.0.0-Java-1.8.eb:36]: 1.8 is also the old oracle variant we should avoid using and at least favor `('Java', '8'),`

Comment (boegel) [reply easybuild/easyconfigs/s/SNAP-ESA/SNAP-ESA-9.0.0-Java-1.8.eb:36]: @lexming fixed in 8fbc32e6e2

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Review by lexming — APPROVED** [review]
LGTM

**Pull request merged**